docker-gc

```diff
@@ -241,11 +242,23 @@ xargs -n 1 $DOCKER inspect -f '{{.Image}}' 2>/dev/null |
 sort | uniq > images.used
 
 # List images to reap; images that existed last run and are not in use.
-$DOCKER images -q --no-trunc | sort | uniq > images.all
+echo -n "" > images.all
+$DOCKER images | while read line
+do
+    awk '{print $1};'
+done | sort | uniq | while read line
+do
+  $DOCKER images --no-trunc --format "{{.ID}} {{.CreatedAt}}" $line \
+    | sort --key 2 --reverse \
+    | tail --lines +$((MINIMUM_IMAGES_TO_SAVE+1)) \
+    | cut --fields 1 --delimiter " " \
+    | sort \
+    | uniq >> images.all
```

Comment:
Yup that sort was definitely redundant. I've pushed https://github.com/spotify/docker-gc/pull/162/commits/c6da794f54193e07c79d17d26695ec9f7081f1f2 removing it